backend/router/AccountRouterFactory.ts

```diff
@@ -0,0 +1,152 @@
+import { Router } from 'express';
+import { RouterFactory } from './RouterFactory';
+import { AccountController } from '../controller/AccountController';
+import { UserClass } from '../../model/user/IUser';
+import {
+    LoernwerkErrorCodes,
+    requireAdmin,
+    requireBody,
+    requireLogin,
+} from '../loernwerkUtilities';
+
+/**
+ * Builds router for request regarding Account management
+ */
+export class AccountRouterFactory extends RouterFactory {
+    /**
+     * Builds router for Account management
+     * @returns built router
+     */
+    public buildRouter(): Router {
+        const accountRouter = Router();
+
+        accountRouter.post(
+            '/login',
+            requireBody('usernameOrEmail', 'password'),
+            async (req, res) => {
```

Comment:
Do these names have to be `req` and `res` or could they be named `request` and `responese`?

Reply:
Can be changed, `req` and `res` are the express standard though

Reply:
We would need to debate between standard and redability here. Not only are abbreviations not reccomended in code, in addition these are very similar abbreviations, so there could be confusion. 
I would suggest changing them

Reply:
I don't think these abbreviations are confusing for redability here. I thinks its easier to read abbreviations in this case and if they are express standard, I would leave them in code. But its a subjective question and if you think its better, we can change them

Reply:
I can say from my personal experience just reading the code here, that I was confused why the request made the response a couple of times.

And avoiding abbreviations is just often a good rule when coding in my opinion

Reply:
While I do agree with avoiding abreviations, I think complying with wide-known standards is almost equally important - express themselves use these abbreviations all over their documentation and guides, as well as most the majority of libraries out there, including e.g. `express-session`

Reply:
If you both think its the way to go